lib/auth.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
```

Comment:
Some comments in the header and for the `checkPermissions` fucntion would be desirable.